holoviews/element/raster.py

```diff
@@ -644,6 +421,8 @@ class RGB(Image):
         If an alpha channel is supplied, the defined alpha_dimension
         is automatically appended to this list.""")
 
+    _vdim_reductions = {1: Image}
```

Comment:
Actually it's a bit of pain to replace this, I can file a comment on the issue about this to either generalize this approach in some way or replace it.